src/stim/stabilizers/tableau_pybind_test.py

```diff
@@ -694,6 +694,16 @@ def test_unitary():
     ])
 
 
+def test_to_pauli_string():
+    p = stim.PauliString("+YX_Z")
+    assert p.to_tableau().to_pauli_string() == p
+
+    cnot = stim.Tableau.from_named_gate("CNOT")
+    with pytest.raises(ValueError):
```

Comment:
Add a `matches` parameter to the `raises` block to check the message

Reply:
:+1: